Yatima/Cli/TypecheckCmd.lean

```diff
@@ -1,8 +1,65 @@
 import Cli
+import Yatima.Cli.Utils
 import Yatima.Cli.Version
+import Yatima.Compiler.Compiler
+import Yatima.Typechecker.Typechecker
+
+open Yatima.Typechecker in
+open Yatima.Compiler in
+def typecheckRun (p : Cli.Parsed) : IO UInt32 := do
+  match ← getToolchain with
+  | .error msg => IO.eprintln msg; return 1
+  | .ok toolchain =>
+    if toolchain != Lean.versionString then
+      IO.eprintln
+        s!"Expected toolchain '{Lean.versionString}' but got '{toolchain}'"
+      return 1
+  match p.variableArgsAs? String with
+  | some ⟨args⟩ =>
+    if !args.isEmpty then
+      if !(p.hasFlag "prelude") then setLibsPaths
+      let mut stt : CompileState := default
+      let mut errMsg : Option String := none
+      for arg in args do
+        for filePath in ← getLeanFilePathsList ⟨arg⟩ do
+          match ← compile filePath .false stt with
```

Comment:
Do we want to compile here? I thought we were going to read the store from a `.ya` file (generated from the `compile` command)